### DISCUSSION TITLE
Catch ProcessLookupError / error psutil.NoSuchProces

### DISCUSSION THREAD

**Discussion by Stromer199** [swar/Swar-Chia-Plot-Manager#303] [Q&A]
In viewer I got a ProcessLookupError followed by error psutil.NoSuchProcess a couple times. I guess the plot was done and therefore also the corresponding process was closed. At the same time python was referencing to that no more existing process.
![plotter](https://user-images.githubusercontent.com/68348014/117864694-6340e280-b295-11eb-9a5c-58a2693e19e6.JPG)

**Comment by swar** [comment]
Try using the most recent update of the manager.

Please use GitHub Discussions for questions as this is not an Issue.
